integration_tests/snapshots/logs/process-input-traced_node16.log

```diff
@@ -48,7 +47,6 @@ START
           "_inferred_span.tag_source": "self",
           "_inferred_span.synchronicity": "sync",
           "http.method": "GET",
-          "http.route":"/{proxy+}",
```

Comment:
At the time of creating https://github.com/DataDog/datadog-lambda-js/pull/584
I still had the issue of not being able to update the snapshots. So I added the original lines manually, and I must have put them in the wrong place even though the tests still passed.

It seems like they _should_ be in the invocation span, not the inferred span.

Reply:
I disagree. I think it should be on the inferred span since for an api.gateway span, that's where the idea of the routing is happening. Can we run these again and confirm where these tags are actually being added? This is the only trace out of all the test invocates in which it's been moved.

Reply:
The code that sets the http.route tags is here: https://github.com/DataDog/datadog-lambda-js/blob/56965ac722d419af55608f445e06f5aaadbfd858/src/trace/trigger.ts#L299-L318

If you backtrack the call stack, it starts at `onStartInvocation` and stores the result in `this.triggerTags`, so at least according to the code, we are setting the tags on the Lambda span -- not the API GW inferred span.

More info: https://datadoghq.atlassian.net/browse/SVLS-5779